rackcity/api/serializers/asset_serializers.py

```diff
@@ -342,6 +366,20 @@ def serialize_power_connections(power_port_model, asset):
     return power_connections
 
 
+def get_blades_in_chassis(asset):
+    if asset.model.model_type != ModelType.BLADE_CHASSIS.value:
+        return []
+
+    blades = Asset.objects.filter(chassis=asset.id)
+    serializer = AssetSerializer(blades, many=True,)
+    return serializer.data
+
+
+def get_blades_in_chassis_cp(asset_cp):
+    # TODO
+    return []
```

Comment:
Assuming you have a task for this

Reply:
yeet mental task but i promise i will make a notion task

Reply:
Pinky promise?

Reply:
![image](https://user-images.githubusercontent.com/47611824/79079594-1c716980-7cde-11ea-8a77-699de56191d3.png)

Reply:
:)

Reply:
🥰